jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -1,6 +1,5 @@
 <launch>
   <arg name="hostname" default="fetch15" />
-  <arg name="launch_moveit" default="true" />
```

Comment:
why we move launch_moveit from fetch_bringup.launch to fetch.launch

Reply:
I don't know exactly why @knorth55 has moved it. But it may be because he want to run fetch arm with only fetch.launch.

Reply:
I think without this, we cannot tuck fetch arm when fetch_bringup.launch fails.